File: Platforms/DSSAT_EPICST_standalone/src/java/DSSAT_EPICST_standalone/STEMP_EPIC_Exogenous.java
import  java.io.*;
import  java.util.*;
import java.time.LocalDateTime;

public class STEMP_EPIC_Exogenous
{
    private Double TMAX;
    private Double RAIN;
    private Double TMIN;
    private Double TAVG;
    private Double TAV;
    private Double DEPIR;
    private Double BIOMAS;
    private Double SNOW;
    private Double MULCHMASS;
    private Double TAMP;
    
    public STEMP_EPIC_Exogenous() { }
    
    public STEMP_EPIC_Exogenous(STEMP_EPIC_Exogenous toCopy, boolean copyAll) // copy constructor 
    {
        if (copyAll)
        {
            this.TMAX = toCopy.getTMAX();
            this.RAIN = toCopy.getRAIN();
            this.TMIN = toCopy.getTMIN();
            this.TAVG = toCopy.getTAVG();
            this.TAV = toCopy.getTAV();
            this.DEPIR = toCopy.getDEPIR();
            this.BIOMAS = toCopy.getBIOMAS();
            this.SNOW = toCopy.getSNOW();
            this.MULCHMASS = toCopy.getMULCHMASS();
            this.TAMP = toCopy.getTAMP();
        }
    }
    public Double getTMAX()
    { return TMAX; }

    public void setTMAX(Double _TMAX)
    { this.TMAX= _TMAX; } 
    
    public Double getRAIN()
    { return RAIN; }

    public void setRAIN(Double _RAIN)
    { this.RAIN= _RAIN; } 
    
    public Double getTMIN()
    { return TMIN; }

    public void setTMIN(Double _TMIN)
    { this.TMIN= _TMIN; } 
    
    public Double getTAVG()
    { return TAVG; }

    public void setTAVG(Double _TAVG)
    { this.TAVG= _TAVG; } 
    
    public Double getTAV()
    { return TAV; }

    public void setTAV(Double _TAV)
    { this.TAV= _TAV; } 
    
    public Double getDEPIR()
    { return DEPIR; }

    public void setDEPIR(Double _DEPIR)
    { this.DEPIR= _DEPIR; } 
    
    public Double getBIOMAS()
    { return BIOMAS; }

    public void setBIOMAS(Double _BIOMAS)
    { this.BIOMAS= _BIOMAS; } 
    
    public Double getSNOW()
    { return SNOW; }

    public void setSNOW(Double _SNOW)
    { this.SNOW= _SNOW; } 
    
    public Double getMULCHMASS()
    { return MULCHMASS; }

    public void setMULCHMASS(Double _MULCHMASS)
    { this.MULCHMASS= _MULCHMASS; } 
    
    public Double getTAMP()
    { return TAMP; }

    public void setTAMP(Double _TAMP)
    { this.TAMP= _TAMP; } 
    
}

File: Platforms/DSSAT_EPICST_standalone/src/cs/DSSAT_EPICST_standalone/STEMP_EPIC_Exogenous.cs
using System;
using System.Collections.Generic;

public class STEMP_EPIC_Exogenous 
{
    private double _DEPIR;
    private double _BIOMAS;
    private double _SNOW;
    private double _RAIN;
    private double _TMAX;
    private double _TAV;
    private double _TAMP;
    private double _TMIN;
    private double _MULCHMASS;
    private double _TAVG;
    
        public STEMP_EPIC_Exogenous() { }
    
    
    public STEMP_EPIC_Exogenous(STEMP_EPIC_Exogenous toCopy, bool copyAll) // copy constructor 
    {
    if (copyAll)
    {
    
    _DEPIR = toCopy._DEPIR;
    _BIOMAS = toCopy._BIOMAS;
    _SNOW = toCopy._SNOW;
    _RAIN = toCopy._RAIN;
    _TMAX = toCopy._TMAX;
    _TAV = toCopy._TAV;
    _TAMP = toCopy._TAMP;
    _TMIN = toCopy._TMIN;
    _MULCHMASS = toCopy._MULCHMASS;
    _TAVG = toCopy._TAVG;
    }
    }
    public double DEPIR
        {
            get { return this._DEPIR; }
            set { this._DEPIR= value; } 
        }
    public double BIOMAS
        {
            get { return this._BIOMAS; }
            set { this._BIOMAS= value; } 
        }
    public double SNOW
        {
            get { return this._SNOW; }
            set { this._SNOW= value; } 
        }
    public double RAIN
        {
            get { return this._RAIN; }
            set { this._RAIN= value; } 
        }
    public double TMAX
        {
            get { return this._TMAX; }
            set { this._TMAX= value; } 
        }
    public double TAV
        {
            get { return this._TAV; }
            set { this._TAV= value; } 
        }
    public double TAMP
        {
            get { return this._TAMP; }
            set { this._TAMP= value; } 
        }
    public double TMIN
        {
            get { return this._TMIN; }
            set { this._TMIN= value; } 
        }
    public double MULCHMASS
        {
            get { return this._MULCHMASS; }
            set { this._MULCHMASS= value; } 
        }
    public double TAVG
        {
            get { return this._TAVG; }
            set { this._TAVG= value; } 
        }
}